forum/qa-content/qa-page.js

```diff
@@ -353,17 +618,24 @@ function qa_ajax_error()
 				modal.appendChild(modalContent);
 				modalParent.appendChild(modal);
 				
+				// when SyntaxHighlighter is available (it's not on /ask site)
+				if (window.hasOwnProperty('SyntaxHighlighter'))
+				{
+					// use forum built-in plugin to make Modal blocks of code highlighted
+					SyntaxHighlighter.highlight();
+				}
+				
 				/*
 				 * prepare blocks of code inside Preview to be collapsed/expanded
 				 * "true" parameter lets to display collapsing blocks inside Preview Modal
 				 */
-				handleCodeCollapsing(true);
+				handleCodeCollapsing(true, prepareClipboardCopying);
 			}
 		});
-	}
+	};
 	
 	// when Forum (sub)page DOM is ready

```

Comment:
I would change this comment to:

> // when Forum (sub)page DOM with it's CSSes and synchronously loaded scripts (excluding CKEDITOR, which needs separate Event Handling) are ready

, because a little bit later, there is `CKEDITOR.on('instanceReady', function(){});` EventHandler - and mentioned comment describes that despite the `load` Event handles page load, CKEDITOR on it's own is loaded asynchronous and have to be served separately - as @Comandeer mentioned: https://github.com/CodersCommunity/forum.pasja-informatyki.local/issues/15#issuecomment-230801504